pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -316,6 +352,33 @@ var _ = Describe("domain network interfaces resources", func() {
 type libvirtClientResult struct {
 	expectedError           error
 	expectedAttachedDevices int
+	expectedUpdatedDevices  int
+	expectedArgument        string
+}
+type containsMatcher struct {
+	substr string
+}
```

Comment:
Sorry, but I am not interested in maintaining this sophisticated matcher, unless you give me a very strong reason.
What are the alternatives? If it is to include the whole interface config, do that please... it is both easier to read and avoid this smart code.